apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/CreateProfile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -50,20 +51,37 @@ const PersonalInfo = () => {
             maxLength={40}
           />
         </InputFormError>
-        <div className={styles.emailContainer}>
+        <TextInput
+          label={t('email')}
+          value={session?.user?.email || ''}
+          onChange={handleChange}
+          addImportantIcon={true}
+          name={PersonalInfoFormKeys.CONTACT_EMAIL}
+          disabled={true}
+          tooltipText={t('emailTooltip')}
+          dataTestId=""
+          maxLength={30}
+        />
+        <UserPhotoUploader />
+        <InputFormError
+          error={
+            touched[PersonalInfoFormKeys.SLUG] &&
+            errors[PersonalInfoFormKeys.SLUG]
+          }
+        >
           <TextInput
-            label={t('email')}
-            value={session?.user?.email || ''}
-            onChange={handleChange}
             addImportantIcon={true}
-            name={PersonalInfoFormKeys.CONTACT_EMAIL}
-            disabled={true}
-            tooltipText={t('emailTooltip')}
-            dataTestId=""
-            maxLength={30}
+            tooltipText={t('slugTooltip')}
+            onBlur={handleBlur}
```

Comment:
**issue:** We should validate this field on blur this way, so it checks in DB is such slug is not already taken. This validation is crucial and we should do it sooner than after form submission